main.go

```diff
@@ -116,10 +118,10 @@ func registerResources() {
 	didPool.SetRegistrar(regInstance)               //Registering telephony did pools
 	archIvr.SetRegistrar(regInstance)               //Registering architect ivr
 	workbin.SetRegistrar(regInstance)               //Registering task management workbin
+	team.SetRegistrar(regInstance)                  // Addomg tea, register
```

Comment:
comment is off here

Reply:
comment addressed but it has not updated here.